src/QMCWaveFunctions/TrialWaveFunction.h

```diff
@@ -144,11 +144,11 @@ class TrialWaveFunction : public MPIObjectBase
 
   /** functions to handle particle-by-particle update */
   RealType ratio(ParticleSet& P, int iat);
-  ValueType full_ratio(ParticleSet& P, int iat);
+  ValueType evaluateFullRatio(ParticleSet& P, int iat);
```

Comment:
Adding a documentation for this function and note the difference to ratio

Reply:
I have added documentation for this function.